Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -403,6 +403,41 @@ - (void)testLinkUser {
 
 }
 
+#pragma mark - Auth Credentials
+
+- (void)testAuthCredentials {
+    RLMAppCredentials *usernamePasswordCredential = [RLMAppCredentials credentialsWithUsername:@"test@mongodb.com" password:@"apassword"];
+    XCTAssertEqualObjects(usernamePasswordCredential.provider, @"local-userpass");
```

Comment:
[nit] each one of these can be a separate test

Reply:
done